scripts/code-coverage

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+DIR=$1
+
+# Delete existing coverage data
+echo "Deleting previous coverage information..."
+find "$DIR" -type f -name '*.gcda' -delete
+
+# Create zero coverage data
+echo "Generating coverage baseline..."
+lcov -d "$DIR" -c -i -o "$DIR/oeedger8r_base.info" \
```

Comment:
I assume we this only works for gcc. Should we document this or add check for compilers?

Reply:
Good point. Added cmake check for GCC.